com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -149,29 +149,32 @@ private static void initAndroid() {
                 Bob.atomicCopy(classes_dex, f, false);
             }
 
-            // Make sure it's extracted once
-            File bundletool = new File(Bob.getLibExecPath("bundletool-all.jar"));
+            String aapt2 = Bob.getExe(Platform.getHostPlatform(), "aapt2");
+            if (aapt2 == null) {
+                // Make sure it's extracted once
+                File bundletool = new File(Bob.getLibExecPath("bundletool-all.jar"));
 
-            // Find the file to extract from the bundletool
-            {
-                Platform hostPlatform = Platform.getHostPlatform();
-                String platformName = "macos";
-                String suffix = "";
-                if (hostPlatform == Platform.X86_64Win32)
-                {
-                    platformName = "windows";
-                    suffix = ".exe";
-                }
-                else if (hostPlatform == Platform.X86_64Linux)
+                // Find the file to extract from the bundletool
                 {
-                    platformName = "linux";
-                }
+                    Platform hostPlatform = Platform.getHostPlatform();
```

Comment:
Seems it worth to move outside condition body and use it in `String aapt2 = Bob.getExe(Platform.getHostPlatform(), "aapt2");`